### ISSUE TITLE
Not able to install

### ISSUE THREAD

**Issue by BerT666** [nextcloud/contacts#37]
### Steps to reproduce
1. download via NextCloud AppStore 
2. getting error Message (not installable)
3. What I found when checking:
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- contacts
	- EXCEPTION
		- OC\IntegrityCheck\Exceptions\InvalidSignatureException
		- Signature data not found.

Raw output
==========
Array
(
    [contacts] => Array
        (
            [EXCEPTION] => Array
                (
                    [class] => OC\IntegrityCheck\Exceptions\InvalidSignatureException
                    [message] => Signature data not found.
                )

        )

)

### Expected behaviour
The Applet should install ;-)

### Actual behaviour
Installation failed

### Server configuration
**Operating system**: Ubuntu Server 16.04 LTS

**Web server:** Apache

**Database:** mysql 10.0.27

**PHP version:** 7.0.8

**Nextcloud version:** (see Nextcloud admin page) 10.0.1 (stable)

**Contacts version:** (see Nextcloud apps page) -/- tried install via Store and it fails (Ver 1.5.0)

**Updated from an older Nextcloud or fresh install:** fresh install

**Signing status:**
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- contacts
	- EXCEPTION
		- OC\IntegrityCheck\Exceptions\InvalidSignatureException
		- Signature data not found.

Raw output
==========
Array
(
    [contacts] => Array
        (
            [EXCEPTION] => Array
                (
                    [class] => OC\IntegrityCheck\Exceptions\InvalidSignatureException
                    [message] => Signature data not found.
                )

        )

)

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**
Calendar LDAP 
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
Not yet ;-)

**Are you using encryption:** yes

**Are you using an external user-backend, if yes which one:** ActiveDirectory (also happens with local nextcloud user)

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:** tried with latest Chrome IE 11 and Edge (all different configurations)

**Operating system:** Win7 / Win 10 / Ubuntu 14.04 Desktop

**CardDAV-clients:**

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by pierre-alain-b** [comment]
Same issue here. Unable to install 1.5.0.
After the installation attempt, the following error message can be seen:
"Signature could not get checked. Please contact the app developer and check your admin screen."

**Comment by anmith** [comment]
same here.

**Comment by nickvergessen** [comment]
@irgendwie you forgot to sign the package, before uploading it

**Comment by Akhenaton** [comment]
Solved with 1.5.2 Thank you :thumbsup:
